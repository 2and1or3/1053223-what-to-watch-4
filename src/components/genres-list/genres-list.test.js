import React from "react";
import renderer from "react-test-renderer";

import GenresList from './genres-list.jsx';

it(`Render GenresList component`, () => {
  const tree = renderer
    .create(<GenresList currentGenre = {`all`} onLinkClick = {() => {}}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
