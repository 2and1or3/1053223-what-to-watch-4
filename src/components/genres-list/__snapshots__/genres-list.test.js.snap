// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render GenresList component 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item catalog__genres-item--active"
    id="all"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      All genres
    </a>
  </li>
  <li
    className="catalog__genres-item "
    id="comedy"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Comedies
    </a>
  </li>
  <li
    className="catalog__genres-item "
    id="crime"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Crime
    </a>
  </li>
  <li
    className="catalog__genres-item "
    id="documentary"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Documentary
    </a>
  </li>
  <li
    className="catalog__genres-item "
    id="drama"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Dramas
    </a>
  </li>
  <li
    className="catalog__genres-item "
    id="horror"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Horror
    </a>
  </li>
  <li
    className="catalog__genres-item "
    id="family"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Kids & Family
    </a>
  </li>
  <li
    className="catalog__genres-item "
    id="romance"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Romance
    </a>
  </li>
  <li
    className="catalog__genres-item "
    id="sciFi"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Sci-Fi
    </a>
  </li>
  <li
    className="catalog__genres-item "
    id="thriller"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Thrillers
    </a>
  </li>
</ul>
`;
